cumulus/test/service/src/cli.rs

```diff
@@ -60,23 +60,23 @@ pub enum Subcommand {
 	BuildSpec(sc_cli::BuildSpecCmd),
 
 	/// Export the genesis state of the parachain.
-	ExportGenesisState(ExportGenesisStateCommand),
+	ExportGenesisState(ExportGenesisHeadCommand),
 
 	/// Export the genesis wasm of the parachain.
 	ExportGenesisWasm(ExportGenesisWasmCommand),
 }
 
 #[derive(Debug, clap::Parser)]
 #[group(skip)]
-pub struct ExportGenesisStateCommand {
+pub struct ExportGenesisHeadCommand {
 	#[arg(default_value_t = 2000u32)]
 	pub parachain_id: u32,
```

Comment:
I don't remember why we have `parachain_id` here, it looks like it is unused. Since we are at it, you could clean this up if you like.

Reply:
I was already feeling a sloppy about doing both the rename and the fix in a single PR. I kind of prefer a separate issue for this.

I also hesitate a little bit because I'm not sure if removing this will break zombienet. And I also don't know the history of when it stopped being used.

Reply:
Okay, I will create an issue and can assign it to myself, will come back to this.